conf/environment.yml

```diff
@@ -2,7 +2,7 @@ name: yosys-env
 channels:
   - symbiflow
 dependencies:
-  - symbiflow-yosys
+  - symbiflow-yosys=0.8_6021_gd8b2d1a2=20200923_144930*
```

Comment:
Can you log an issue to remove this version specification?

Reply:
Actually I put the version here on purpose so that whenever the a new Conda package gets uploaded it won't break V2X docs build system. So I'd like to have the version fixed.

Thoughts?

Reply:
Removed explicit version specification.